connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaOffsetBackingStoreTest.java

```diff
@@ -105,33 +112,58 @@ public class KafkaOffsetBackingStoreTest {
 
     @Mock
     KafkaBasedLog<byte[], byte[]> storeLog;
+    @Mock
+    Supplier<TopicAdmin> topicAdminSupplier;
     private KafkaOffsetBackingStore store;
 
-    private Capture<String> capturedTopic = EasyMock.newCapture();
-    private Capture<Map<String, Object>> capturedProducerProps = EasyMock.newCapture();
-    private Capture<Map<String, Object>> capturedConsumerProps = EasyMock.newCapture();
-    private Capture<Supplier<TopicAdmin>> capturedAdminSupplier = EasyMock.newCapture();
-    private Capture<NewTopic> capturedNewTopic = EasyMock.newCapture();
-    private Capture<Callback<ConsumerRecord<byte[], byte[]>>> capturedConsumedCallback = EasyMock.newCapture();
+    private MockedStatic<WorkerConfig> workerConfigMockedStatic;
+
+    @Captor
+    private ArgumentCaptor<String> capturedTopic;
+    @Captor
+    private ArgumentCaptor<Map<String, Object>> capturedProducerProps;
+    @Captor
+    private ArgumentCaptor<Map<String, Object>> capturedConsumerProps;
+    @Captor
+    private ArgumentCaptor<Supplier<TopicAdmin>> capturedAdminSupplier;
+    @Captor
+    private ArgumentCaptor<NewTopic> capturedNewTopic;
+    @Captor
+    private ArgumentCaptor<Callback<ConsumerRecord<byte[], byte[]>>> capturedConsumedCallback;
+    @Captor
+    private ArgumentCaptor<Callback<Void>> storeLogCallbackArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<Callback<Void>> secondGetReadToEndCallback;
```

Comment:
Probably makes sense to inline this one since it's only used in one test.

Reply:
Actually, would it be possible to replace uses of it altogether with the `storeLogCallbackArgumentCaptor` field?